src/app/containers/Include/index.jsx

```diff
@@ -43,7 +43,7 @@ const IncludeContainer = props => {
     return (
       <FallbackGrid>
         <EmbedError
-          message="Sorry, we can’t display this part of the story on this lightweight mobile page."
+          message="Sorry, we can’t display this part of the story on this lightweight mobile page. "
```

Comment:
Would it be better to apply the space in the code? This could get lost here when translations are added?

Reply:
Yes it would be. I think we should merge in this PR and raise a ticket for this issue.

Reply:
Ticket raised: https://github.com/bbc/psammead/issues/3625